### PR TITLE
Change getnotificationsmethod opcode

### PR DESCRIPTION
**Pull request by luc10921**
**Summary or solution description**
Changed the method arg and the default value

**How to Reproduce**
https://github.com/CityOfZion/neo3-boa/blob/fc773d29aa2d4eedf7dc019efc7f71e6337b4036/boa3_test/test_sc/interop_test/runtime/GetNotifications.py#L1-L22

**Tests**
https://github.com/CityOfZion/neo3-boa/blob/fc773d29aa2d4eedf7dc019efc7f71e6337b4036/boa3_test/tests/compiler_tests/test_interop/test_runtime.py#L427-L469

**Platform:**
 - OS: Windows 10 x64
 - Python version: Python 3.7

### CODE REVIEW

**Comment by lllwvlvwlll** [comment]
Task linked: [CU-2kq1zbu Change getnotificationsmethod opcode](https://app.clickup.com/t/2kq1zbu)

File: boa3_test/tests/compiler_tests/test_interop/test_runtime.py [hunk 1]
```diff
@@ -426,20 +426,30 @@ def test_invocation_counter_cant_assign(self):
...
-
         engine = TestEngine()
+
+        from boa3.constants import MANAGEMENT_SCRIPT
+
         result = self.run_smart_contract(engine, path, 'without_param', [])
-        self.assertEqual([], result)
+        self.assertEqual(1, len(result))
+        self.assertEqual(MANAGEMENT_SCRIPT, result[0][0])
+        self.assertEqual('Deploy', result[0][1])
```
Comment (meevee98) [inline boa3_test/tests/compiler_tests/test_interop/test_runtime.py:436]: import `boa3.constants` in the beginning of the file and use `constants.MANAGEMENT_SCRIPT` here
also use variables no make the code clearer
```suggestion
        self.assertEqual(1, len(result))
        self.assertEqual(2, len(result[0]))
        event_script, event_name = result[0][:1]
        self.assertEqual(constants.MANAGEMENT_SCRIPT, event_script)
        self.assertEqual('Deploy', event_name)
```

File: boa3_test/tests/compiler_tests/test_interop/test_runtime.py [hunk 2]
```diff
@@ -426,20 +426,30 @@ def test_invocation_counter_cant_assign(self):
...
-            expected_result.append([script,
-                                    'notify',
-                                    [x]])
-        self.assertEqual(expected_result, result)
+        expected_result = [
+            [MANAGEMENT_SCRIPT, 'Deploy', script],
+            [script, 'notify', [1]],
+            [script, 'notify', [2]],
+            [script, 'notify', [3]],
+        ]
```
Comment (meevee98) [inline boa3_test/tests/compiler_tests/test_interop/test_runtime.py:446]:
```suggestion
        expected_result = [ [MANAGEMENT_SCRIPT, 'Deploy', script] ]
        for x in [1, 2, 3]:
            expected_result.append([script, 'notify', [x]])
        
        self.assertEqual(expected_result, result)
```

File: boa3_test/tests/compiler_tests/test_interop/test_runtime.py [hunk 3]
```diff
@@ -426,20 +426,30 @@ def test_invocation_counter_cant_assign(self):
...
+
+        # the Deploy parameter should have been the smart contract address, but the deploy method does uses a
+        # ReferenceCounter and the test engine didn't replicate this behavior
+        # new VM.Types.Array(engine.ReferenceCounter) { contract.Hash.ToArray() }
+        self.assertEqual((expected_result[0][0], expected_result[0][1]), (result[0][0], result[0][1]))
```
Comment (meevee98) [inline boa3_test/tests/compiler_tests/test_interop/test_runtime.py:451]:
```suggestion
        self.assertEqual(expected_result[0][:1], result[0][:1])
```
increase readability

**Pull request merged**